src/components/List/List.stories.tsx

```diff
@@ -113,3 +159,53 @@ export const cardList = () => (
     </List>
   </>
 );
+/* eslint-disable jsx-a11y/anchor-is-valid */
+export const listOfLinks = () => (
+  <List title="Details" type={ListTypes.Definition}>
+    <dt>Authors</dt>
+    <dd>
+      <a href="#">Chirwa, Ephraim Wadonda, author</a>
+      <br />
```

Comment:
does this mean that the correct way to implement a definition list within design system is to use brs between every link?  Is it against standards to do 
```
  <dt>Authors</dt>
    <dd>
      <li><a href="#">Chirwa, Ephraim Wadonda, author</a></li>
     <li><a href="#">Dorward, Andrew, author</a></li>
```

Or, is it required that `dl` should only be used for definitions the way that tables shouldn't ever be used for anything except tabular data, and therefore, is this not the correct component for this use case?

Reply:
The `<br />` doesn't seem right and maybe can be fixed with CSS. Your example is somewhat correct, it just needs an `<ul>`:

```
<dt>Authors</dt>
<dd>
  <u>
    <li><a href="#">Chirwa, Ephraim Wadonda, author</a></li>
    <li><a href="#">Dorward, Andrew, author</a></li>
  </ul>
</dd>
```

Reply:
You can basically put any markup you want inside the DD.  I wasn't advocating BRs by including them in the example.   The UL suggestion is a good one, or you could also use CSS to set the links to `display: block` and use margin to hone the spacing between them.